src/test/java/org/prebid/server/analytics/reporter/greenbids/GreenbidsAuctionContext.java

```diff
@@ -0,0 +1,109 @@
+package org.prebid.server.analytics.reporter.greenbids;
+
+import com.iab.openrtb.request.Banner;
+import com.iab.openrtb.request.BidRequest;
+import com.iab.openrtb.request.Format;
+import com.iab.openrtb.request.Imp;
+import com.iab.openrtb.request.Site;
+import com.iab.openrtb.response.Bid;
+import com.iab.openrtb.response.BidResponse;
+import com.iab.openrtb.response.SeatBid;
+import org.prebid.server.auction.model.AuctionContext;
+import org.prebid.server.auction.model.BidRejectionReason;
+import org.prebid.server.auction.model.BidRejectionTracker;
+import org.prebid.server.model.HttpRequestContext;
+
+import java.math.BigDecimal;
+import java.util.Collections;
+import java.util.Set;
+
+public class GreenbidsAuctionContext {
```

Comment:
do not need to have a separate class for the stuff that used only in the single test class